node/internal.ts

```diff
@@ -426,7 +427,7 @@ export function _which(tool: string, check?: boolean): string {
 function _tryGetExecutablePath(filePath: string, extensions: string[]): string {
     try {
         // test file exists
-        let stats: fs.Stats = fs.statSync(filePath);
+        let stats: fs.Stats = fs.statSync(_unTildify(filePath));
```

Comment:
Since untildify is only run on that path that is stat'd and not the path that is returned, wouldn't which ultimately return the "wrong" tildified path?